src/database/migrations/20230314092003-add_mfa_code_to_users_table.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable arrow-body-style */
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    return queryInterface.sequelize.transaction((t) => {
+      return Promise.all([
+        queryInterface.addColumn(
```

Comment:
can you add those columns in `20230308210129-create-users.js` migration file instead of adding other migration file